src/reducers/sources.js

```diff
@@ -8,10 +8,10 @@
  * @module reducers/sources
  */
 
-const I = require("immutable");
-const makeRecord = require("../utils/makeRecord");
-const { getPrettySourceURL } = require("../utils/source");
-const { prefs } = require("../utils/prefs");
+import * as I from "immutable";
```

Comment:
Oh that's interesting.  I wonder if this means we should only be importing the things we need from `immutable` in the future.  Thanks for pointing this out and looking it up. 🥂